tests/docker-images/latest-version-image/conf/functions_worker.conf

```diff
@@ -22,7 +22,7 @@ autostart=false
 redirect_stderr=true
 stdout_logfile=/var/log/pulsar/functions_worker.log
 directory=/pulsar
-environment=PULSAR_MEM="-Xmx128M -XX:+HeapDumpOnOutOfMemoryError -XX:HeapDumpPath=/pulsar/logs/functions",PULSAR_GC="-XX:+UseZGC"
```

Comment:
I don't think this is needed in 3.1 but please double-check.
The problem with 2.10 is that it runs with jdk 11 where ZGC isn't enabled by default and in experimental version.
3.1 runs with JDK 17 IIRC and this should be supported.